src/components/v1/account/services/createAccount.js

```diff
@@ -9,9 +9,11 @@ const createAccount = async ({
   contactInformation,
   active,
 }) => {
-  const validateUniqueAccount = await accountDao.findAccountByName({ name });
+  const validateUniqueAccount = await accountDao.findAccountByName(name);
 
-  if (validateUniqueAccount) { throw boom.badRequest(`Account: ${name} already exists`); }
+  if (validateUniqueAccount) {
+    throw boom.badRequest(`Account: ${name} already exists`);
```

Comment:
For next commits, this:
`if (validateUniqueAccount) throw boom.badRequest(`Account: ${name} already exists`);`

Instead that

Reply:
noted